src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -6339,7 +6339,7 @@ private void SNISSPIData(byte[] receivedBuff, UInt32 receivedLength, ref byte[]
                 {
                     _physicalStateObj.GenerateSspiClientContext(receivedBuff, receivedLength, ref sendBuff, ref sendLength, _sniSpnBuffer);
                 }
-                catch (Exception e)
+                catch (InvalidOperationException e)
```

Comment:
Why did this change?  Previously it was catching everything and routing it through SSPIError; now it's only catching InvalidOperationExceptions.

Reply:
i agree with you, revert.